Bender.lock

```diff
@@ -34,6 +34,13 @@ packages:
     - common_verification
     - register_interface
     - tech_cells_generic
+  axi_mem_if:
```

Comment:
This is added because the dependency is in `pulp-ethernet`, please remove it as from my review (https://github.com/pulp-platform/pulp-ethernet/pull/3/files#r1500435712), it should not be needed.